index.html

```diff
@@ -566,8 +587,134 @@ <h1>Trust Model</h1>
 
     <section>
       <h1>Basic Concepts</h1>
+
+      <section>
+        <h2>Types</h2>
+
+        <p>
+Type information may be expressed via the <a>type</a> property:
+        </p>
+        <dl>
+          <dt><dfn>type</dfn></dt>
+          <dd>
+The value of this property MUST be one or more URIs.
+If more than one URI is provided, the array MUST be interpreted as a set of
+values where order is not significant. Note that syntax such as JSON-LD terms
+MAY be used to ease developer usage. It is RECOMMENDED that dereferencing the
+URI results in a document containing machine-readable information about the
+type.
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of the type property">
+{
+  "id": "http://dmv.example.gov/credentials/3732",
+  <span class="highlight">"type": ["VerifiableCredential", "ProofOfAgeCredential"]</span>,
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "ageOver": 21
+  },
+  "proof": { ... }
+}
+        </pre>
+
+        <p>
+With respect to this specification, the following <a>type</a>s MUST be
+specified on the following objects:
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Concept</th>
+              <th>Type</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>
+A <a href="#credentials">credential</a> object containing a
+<a href="#proofs-aka-signatures">proof</a>
+              </td>
+              <td>
+<tt>"type": ["<code>VerifiableCredential</code>"]</tt>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+A <a href="#credentials">credential</a> object NOT containing a
+<a href="#proofs-aka-signatures">proof</a>
+              </td>
+              <td>
+<tt>"type": ["<code>UnverifiableCredential</code>"]</tt>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+A <a href="#presentations">presentation</a> object containing a
+<a href="#proofs-aka-signatures">proof</a>
+              </td>
+              <td>
+<tt>"type": ["<code>VerifiablePresentation</code>"]</tt>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+A <a href="#presentations">presentation</a> object NOT containing a
+<a href="#proofs-aka-signatures">proof</a>
+              </td>
+              <td>
+<tt>"type": ["<code>UnverifiablePresentation</code>"]</tt>
```

Comment:
Again, I'm not a fan of this type. How can we make such a claim that it is unverifiable? All we're saying here is that there's no proof attached. Perhaps "Presentation" is better? That doesn't seem to have the same problems that "Credential" has above.

Reply:
Ditto above for this.